### DISCUSSION TITLE
Errant Expression Profiles

### DISCUSSION THREAD

**Discussion by jhunt8** [Open-Systems-Pharmacology/Forum#1606] [Questions & Problems]
Hi All, 

I'm working with a large project file that has several expression profiles. I can't delete two of these expression profiles even after removing them from all populations and individuals: there remains an "invisible" link between these profiles and some populations and individuals. I can't be sure how I accomplished this error in the first place, but it may have been through renaming expression profiles, even though renaming expression profiles hasn't caused errors for me in other projects. 

Recreating the populations and individuals would likely solve this, but I would prefer a more efficient solution to scrub these errant profiles from the project? Many thanks!

**Comment by PavelBal** [comment]
This seems to be a bug in the software. Would it be possible for you to share your project for debugging?

**Comment by jhunt8** [comment]
We plan to share the project file as part of a publication soon. To ensure that we can submit the content to the journal as "not published elsewhere", can I email you the file directly? Thank you for your help!

**Comment by PavelBal** [comment]
@msevestre @Yuri05 Could you maybe look into it? Sounds like a bug.

**Reply by Yuri05** [reply]
hard to say without the project

**Comment by Yuri05** [comment]
@jhunt8 If you export your project to snapshot (.json) and open it (e.g. with a text editor): do you see any references to your expression profiles **outside** the `ExpressionProfiles `node? E.g. like highlighted lines in the example below?

<img width="350" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/b54e304a-a90e-4973-b44e-45cc8c8afff5">

**Comment by msevestre** [comment]
I am going to convert this to an issue . @jhunt8 sent me the file
